package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fusliez-notes",
-  "version": "0.8.1",
+  "version": "0.8.2",
```

Comment:
please make sure to use npm version "major"|"minor"|"patch" to update the version, it will create a new tag.

`major.minor.patch`

```npm version "patch"`